src/Common/Infrastructure/BackendResponse.cs

```diff
@@ -48,6 +53,14 @@ public void UpdateNextPageUrlWithQueryParameters(NonPagedQueryCollection request
                 RegexOptions.IgnoreCase);
         }
 
+        private bool ParameterIsAllowed(KeyValuePair<string, StringValues> parameter)
+        {
+            if (IsAtomContent)
```

Comment:
I had to think about it, why we wouldn't do it for from parameter.
I know now, because we have been working on it recently, but maybe add a comment why we don't so we don't forget when we revisit this in the future?

Reply:
good point